lib/redis_memo/memoize_query/cached_select.rb

```diff
@@ -313,9 +313,10 @@ def self.extract_bind_params_recurse(node)
       bind_params
     when Arel::Nodes::Grouping
       # Inline SQL
-      return if node.expr.is_a?(Arel::Nodes::SqlLiteral)
-
+      return bind_params if is_comparison_expr(node.expr)
```

Comment:
before,  when an expr is a Arel::Nodes::SqlLiteral, we'd reject the query.

Currently, we accept the query it the string literal contains `comparison` operator. (note there are edge cases such as "a = '<='")

when the expr does not have a comparison operator, we should still reject the query instead of continuing.

Reply:
ah good catch, thanks